guides/v2.1/javascript-dev-guide/widgets/widget_password_strength_indicator.md

```diff
@@ -0,0 +1,115 @@
+---
+group: javascript-developer-guide
+subgroup: 3_Widgets
+title: PasswordStrengthIndicator widget
+---
+
+## Overview {#password_strength_indicator_overview}
+
+Magento password strength indicator {% glossarytooltip f0dcf847-ce21-4b88-8b45-83e1cbf08100 %}widget{% endglossarytooltip %} show the user immediate feedback on the validity and strength of a password while it is being typed.
+If the password is not strong enough the UI asks the user to choose a new password.
+
+The password strength indicator widget source is [`<Magento_Customer_module_dir>/view/frontend/web/js/password-strength-indicator.js`].
+
+The widget is being used in the following templates:
+
+- [`<Magento_Ui_module_dir>/view/frontend/templates/form/register.phtml`] customer register template.
+- [`<Magento_Ui_module_dir>/view/frontend/templates/form/forgotpassword.phtml`] customer forgot password template.
+- [`<Magento_Ui_module_dir>/view/frontend/templates/form/edit.phtml`] customer account edit template.
+
+## Initialize the password_strength_indicator widget {#password_strength_indicator_initialize}
+
+To initialize the widget in your script, use the following general notation:
+
+```javascript
+$('#password-input').passwordStrengthIndicator({
+    <option1>: <value1>,
+    <option2>: <value2>,
+    ...
+});
+```
+
+For details about how to initialize the widget in a`.phtml` template, refer to the [JavaScript initialization] topic.
```

Comment:
Please add the link to the topic: https://devdocs.magento.com/guides/v2.1/javascript-dev-guide/javascript/js_init.html